examples/frompapers/Izhikevich_2003.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
```

Comment:
Thanks for the PR. Please remove these left-over lines (from the conversion from notebook to plain python) here and also below.

Reply:
Sorry, forget to remove the first comments before pushing it.